src/app/recipes/recipes-cards/recipes-cards.component.html

```diff
@@ -13,7 +13,7 @@ <h3 class="mb-4">{{ 'Recipes.LatestRecipes' | translate }}</h3>
 
 <div
 	class="d-flex justify-content-center mt-3"
-	*ngIf="(recipesAreLoaded$ | async) === false">
+	*ngIf="(recipesAreLoaded$ | async) !== 'Loaded'">
```

Comment:
In your *ts file you can declare:
`public dataState: typeof DataState = DataState`

and then in the template You can use
`(recipesAreLoaded$ | async) !== dataState.Loaded`